breadth_first_search/bfs.py

```diff
@@ -0,0 +1,43 @@
+edge={}

```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpttqwh6ut/breadth_first_search/bfs.py
+++ b/tmp/tmpttqwh6ut/breadth_first_search/bfs.py
@@ -1,4 +1,5 @@
-edge={}
+edge = {}
+
 
 def addEdges(source, destination):
     if(source not in edge):
```